<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>831</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="parameters">
    <attribute name="title">
     <string>Parameters</string>
    </attribute>
    <widget class="QTextEdit" name="length_km">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bus_number_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>to bus</string>
     </property>
    </widget>
    <widget class="QLabel" name="bus_number_label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>std_type</string>
     </property>
    </widget>
    <widget class="QLabel" name="p_kw_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>from_bus</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QWebView" name="webView_2">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>40</y>
       <width>201</width>
       <height>391</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>http://pandapower.readthedocs.io/en/v1.3.0/_images/line.png</string>
      </url>
     </property>
    </widget>
    <widget class="QLabel" name="q_kvar_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>length_km</string>
     </property>
    </widget>
    <widget class="QLabel" name="bus_number_label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="q_kvar_label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>[km]</string>
     </property>
    </widget>
    <widget class="QComboBox" name="standard_type">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="from_bus">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="to_bus">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="help">
    <attribute name="title">
     <string>Documentation</string>
    </attribute>
    <widget class="QWebView" name="webView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>621</width>
       <height>341</height>
      </rect>
     </property>
     <property name="url">
      <url>
       <string>http://pandapower.readthedocs.io/en/v1.3.0/elements/line.html#line</string>
      </url>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
